radixdlt/src/main/java/com/radixdlt/consensus/Vote.java

```diff
@@ -38,36 +40,44 @@
 
 	@JsonProperty("author")
 	@DsonOutput(Output.ALL)
-	private final EUID author;
+	private final RadixAddress author;
```

Comment:
Don't think this should be RadixAddress which is an application level object

Reply:
Yup, agreed, updated